.github/CODEOWNERS

```diff
@@ -1 +1,5 @@
 * @libero/node-js-reviewers
+.docker/* @libero/infrastructure-reviewers
+.github/* @libero/infrastructure-reviewers
+Dockerfile @libero/infrastructure-reviewers
+Makefile @libero/infrastructure-reviewers
```

Comment:
Reading https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax these should be

```
/.docker/ @libero/infrastructure-reviewers
/.github/ @libero/infrastructure-reviewers
/Dockerfile @libero/infrastructure-reviewers
/Makefile @libero/infrastructure-reviewers
```

Also want `/.dockerignore`. `/.scripts too? (Does that count as a file here? 🤔)

Not sure about `/.dependabot/`...

Reply:
In theory `.scripts` should be a file containing the submodule commit so does count for modifications.

I'll go out on a limb and say `/.dependabot` is developer-oriented like the linting or testing configuration so shouldn't be included. It isn't very language-specific still.